File: exercicios/JavaScript/1042/1042.js
var input = require('fs').readFileSync('stdin','utf8');
var lines = input.split('\n');

var line1 = lines.shift().split(' ');

var a = parseInt(line1[0])
var b = parseInt(line1[1])
var c = parseInt(line1[2])

if(a < b && a < c){
    console.log(a)
    if(b < c){
        console.log(b)
        console.log(c)
    }else{
        console.log(c)
        console.log(b)
    }
}else if(b < c){
    console.log(b)
    if(a < c){
        console.log(a)
        console.log(c)
    }else{
        console.log(c)
        console.log(a)
    }
}else{
    console.log(c)
    if(a < b){
        console.log(a)
        console.log(b)
    }else{
        console.log(b)
        console.log(a)
    }
}

console.log("")
console.log(a)
console.log(b)
console.log(c)

File: exercicios/1042/1042.py
entrada = input().split()

n1 = int(entrada[0])
n2 = int(entrada[1])
n3 = int(entrada[2])

if (n1 > n2) and (n1 > n3):
    d = n1
    if(n2 > n3):
        e = n2
        f = n3
    else:
        e = n3
        f = n2

if(n2 > n1) and (n2 > n3):
    d = n2
    if(n1 > n3):
        e = n1
        f = n3
    else:
        e = n3
        f = n1

if(n3 > n1) and (n3 > n2):
    d = n3
    if(n1 > n2):
        e = n1
        f = n2
    else:
        f = n1
        e = n2
print(f)
print(e)
print(d)
print()
print(n1)
print(n2)
print(n3)
